prow/plank/README.md

```diff
@@ -0,0 +1,40 @@
+# Plank
+
+Plank is the controller that manages the job execution and lifecycle for jobs running in k8s.
+
+### Usage
+```bash
+$ bazel run //prow/cmd/plank -- --help
+```
+
+### Configuration
+Only GCS is supported as the log storage at the moment.
+
+```yaml
+# config.yaml
+
+plank:
+  allow_cancellations: true # abort presubmit ProwJobs and delete their Pods when new instances are triggered for the same PR
```

Comment:
We actually abort the presubmit ProwJob in either case. The difference is just that we delete the pod backing the aborted ProwJob if allow_cancellations is true and we allow the pod to run to completion (but ignore its result) if allow_cancellations is false.